specs/DotNetProjectFile.Analyzers.Specs/DotNetProjectFile.Analyzers.Specs.csproj

```diff
@@ -45,4 +45,9 @@
     <ProjectReference Include="..\..\src\DotNetProjectFile.Analyzers\DotNetProjectFile.Analyzers.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Can we then still test if projects are detected when they are not included as additional files? Or has this an other intended effect?

Reply:
This doesn't change this. This just adds all found csproj files to browse in the project without manually adding them one-by-one to the solution.

If we remove the CompileAction, most tests still fail (as expected).

Reply:
In that case I would go for `<None>` as we do not expect any compilation action based on in, write?